en/docs/learn/self-registration-and-account-confirmation.md

```diff
@@ -220,7 +220,24 @@ Next, you can try out self-registration.
 
 3.  Fill in the user details, provide consent to share the requested
     information and then click **Register**.
+    ![self-signup-form](../assets/img/using-wso2-identity-server/self-signup-form.png) 
+        
+    ??? Abstract "Click here to see how to configure above requested attributes for the self registration" 
+        
+        -   Attributes that shows in the
+        self sign up page are the WSO2 [local dialect's](../../learn/adding-claim-mapping/#add-local-claim) claims which has
```

Comment:
The attributes that show up on the self sign up page are WSO2 [local dialect](../../learn/adding-claim-mapping/#add-local-claim) claims that have the **Supported by Default** configuration enabled.